backend/src/test/java/com/benine/backend/http/CreatePresetHandlerTest.java

```diff
@@ -70,7 +79,6 @@ public void testCameraID() throws Exception{
 
     getHandler().handle(target, requestMock, httprequestMock, httpresponseMock);
 
-    verify(presetController).addPreset(preset);
```

Comment:
I deleted this in my other branch because at the time I thought that the error was in the test, which wasn't correct.